docs/csharp/language-reference/configure-language-version.md

```diff
@@ -1,14 +1,19 @@
 ---
 title: C# language versioning - C# Guide
-description: Learn about how the C# language version is determined based on your project, and the different values you can manually adjust it to.
-ms.date: 07/10/2019
+description: Learn about how the C# language version is determined based on your project and the reasons behind that choice. Learn how to override the default manually.
+ms.date: 02/21/2020
 ---
 
 # C# language versioning
 
-The latest C# compiler determines a default language version based on your project's target framework or frameworks. This is because the C# language may have features that rely on types or runtime components that are not available in every .NET implementation. This also ensures that for whatever target your project is built against, you get the highest compatible language version by default.
+The latest C# compiler determines a default language version based on your project's target framework or frameworks. Visual Studio doesn't provide a UI to change the value, but you can change it by editing the *csproj* file. The choice of default ensures that you use the latest language version compatible with your target framework. You benefit from access to the latest language features compatible with your project's target. This default choice also ensures you don't use a language that requires types or runtime behavior not available in your target framework. Choosing a language version newer than the default can cause hard to diagnose compile-time and runtime errors.
 
-The rules in this article apply to the compiler delivered with Visual Studio 2019 or the .NET Core 3.0 SDK. The C# compilers that are part of the Visual Studio 2017 installation or earlier .NET Core SDK versions target C# 7.0 by default. 
+These new defaults mean C# 8.0 (and higher) is supported only on .NET Core 3.x and newer versions. Many of the newest features require library and runtime features introduced in .NET Core 3.x:
```

Comment:
The choice of default doesn't determine what is supported, what is fully supported determines the choice of default.

```suggestion
These criteria mean that C# 8.0 (and higher) is the default language version only on .NET Core 3.x and newer versions. Many of the newest features require library and runtime features introduced in .NET Core 3.x:
```
Or omit "These criteria mean that":

> C# 8.0 (and higher) is the default language version only on .NET Core 3.x and newer versions. Many of the newest features require library and runtime features introduced in .NET Core 3.x:

Reply:
I omitted "these criteria..."